functions/supplier/zeromq-supplier/src/main/java/org/springframework/cloud/fn/supplier/zeromq/ZeroMqSupplierProperties.java

```diff
@@ -64,7 +63,6 @@
 	/**
 	 * @param socketType the {@link SocketType} to establish.
 	 */
-	@NotNull(message = "'socketType' is required")
```

Comment:
You probably wanted to move this onto getter. 
The point is that property like this `zeromq.supplier.socket-type=` is not acceptable.
Same for `consumeDelay`